values.yaml

```diff
@@ -1,12 +1,22 @@
 global:
   labels: {}
   annotations: {}
+  fullNameOverride: "-"
```

Comment:
I know we want to be able to turn this off, but defaulting it off is that the best idea? I feel like that's going to cause chart authors to miss things like prefix's.

If we do this how does someone enable it w/o setting it. Set it to "" or Nil?

Reply:
Ignore me, I've just realized what repo this is, this is fine... :)